CHANGELOG.md

```diff
@@ -28,6 +28,20 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-0284][]:
+
+  Functions, subscripts, and initializers may now have more than one variadic parameter, as long as all parameters which follow variadic parameters are labeled. This makes declarations like the following valid:
+
+  ```swift
+  func foo(_ a: Int..., b: Double...) { }
+
+  struct Bar {
+    subscript(a: Int..., b b: Int...) -> [Int] { a + b }
```

Comment:
Why are there two bs?

Reply:
subscript parameters are unlabeled by default, so the second parameter needs an explicit label since it follows a variadic parameter